views/timelisting.tmpl

```diff
@@ -17,61 +17,50 @@
-                      {{if eq $dropVal $count 20 30 50 100}}{{else}}<option value="{{$dropVal}}">{{$dropVal}} per page</option>{{end}}
-                      {{if eq $count 20 30 50 100 200}}{{else}}<option selected value="{{$count}}">{{$count}} per page</option>{{end}}
+                        <option {{if eq $count 10}}selected{{end}} value="10">10 per page</option>
+                        <option {{if eq $count 20}}selected{{end}} value="20">20 per page</option>
+                        <option {{if eq $count 30}}selected{{end}} value="30">30 per page</option>
+                        <option {{if eq $count 40}}selected{{end}} value="40">40 per page</option>
+                        <option {{if eq $count 50}}selected{{end}} value="50">50 per page</option>
+                        {{if eq $count 10 20 30 40 50}}{{else}}
+                            <option selected value="{{$count}}">{{$count}} per page</option>
+                        {{end}}
                       </select>
                   </span>
-                  <nav aria-label="blocks navigation" data-limit="{{.Limit}}" class="ml-2 my-2 d-inline-block text-right">
-                      <ul class="pages mb-0">
-                          {{if ne .Offset 0}}
-                          <li>
-                              <a
-                              class="text-secondary"
-                              href="/{{$lowerCaseVal}}?offset=0&rows={{.Limit}}"
-                              > Newest</a>
-                          </li>
-                          <li>
-                              <a
-                              class="text-secondary"
-                              href="/{{$lowerCaseVal}}?offset={{subtract .Offset .Limit}}&rows={{.Limit}}"
-                              > Newer</a>
-                          </li>
-                          {{end}}
-                          {{if lt $oldest $lastGrouping}}
-                          <li>
-                              <a
-                              class="text-secondary"
-                              href="/{{$lowerCaseVal}}?offset={{add .Offset .Limit}}&rows={{.Limit}}"
-                              >Older</a>
-                          </li>
-                          <li>
-                              <a
-                              class="text-secondary"
-                              href="/{{$lowerCaseVal}}?offset={{subtract $lastGrouping .Limit}}&rows={{.Limit}}"
-                              >Oldest</a>
-                          </li>
-                          {{end}}
-                      </ul>
-                  </nav>
+                <span class="fs12 nowrap text-right">
+                    <select
+                        data-target="pagenavigation.pageage"
+                        data-action="change->pagenavigation#setPageAge"
+                        data-rows="{{.Limit}}"
+                        data-offsetkey="offset"
```

Comment:
This key is clashing with https://github.com/decred/dcrdata/pull/1577/files#diff-410e820c1016f20eeb74ac755a218848R31, please change it.
Actually you should not use `data-offsetkey` here.

Reply:
this will no longer be needed as it's going back to being buttons